### DISCUSSION TITLE
App Import/Export

### DISCUSSION THREAD

**Discussion by brickaBrackaSN** [Micke-K/IntuneManagement#73] [General]
Hello, upon export I notice I only have a JSON and no .intunewin files. Even if I place these into the Applications folder and ensure the JSON indicates them as the target they don't actually seem to be uploading to another tenant on Import. Is this expected behavior? The result is an object in the applications list which cannot be used and must be deleted:
![image](https://user-images.githubusercontent.com/109824844/180515530-fb19c6a9-0125-4937-9dd3-c7710b285a84.png)

**Comment by brickaBrackaSN** [comment]
Reading through other discussion here I was able to find the answer for any in the same boat. "View" the application and there's an upload button. Additionally in Settings there's a place to declare the application root folder. I'm unable to delete this thread but it can be marked resolved.

Thanks for this amazing tool Micke-K!

**Comment by Micke-K** [comment]
Hello!

Thank you for your support!

You also had the answer in [discussion 58](https://github.com/Micke-K/IntuneManagement/discussions/58) and [discussion 55](https://github.com/Micke-K/IntuneManagement/discussions/55) ;)

Cheers!
